webhook_to_fedora_messaging/models/service.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from sqlalchemy import Column, Integer, UnicodeText, Boolean, ForeignKey
+from webhook_to_fedora_messaging.models.util import UUIDCreatableMixin, CreatableMixin
+
+from webhook_to_fedora_messaging.database import Base
+
+
+class Service(Base, UUIDCreatableMixin, CreatableMixin):
+    __tablename__ = "services"
+
+    id = Column(Integer, primary_key=True, nullable=False)
+    user_id = Column(Integer, ForeignKey("user.id", ondelete="CASCADE"), unique=False, nullable=False)
+    name = Column(UnicodeText, nullable=False)
+    type = Column(UnicodeText, nullable=False)
+    topic = Column(UnicodeText, nullable=False)
```

Comment:
I don't think that a service should be allowed to decide on which topic it's going to be published. I think the app should craft the topic either based on the event name or a combination of the event name and the service name. (I think we currently only use the event name, we should probably keep doing that unless there's a good reason to change).
As a result, I think we can drop this column.